WatchDogCore/WatchDog/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,12 @@
+eclipse.preferences.version=1
```

Comment:
Not really sure, but the folder and the files have also been available in Eclipse plugin.